_posts/2019-10-3-Yong.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: Yong
+---
+
+# **My name is Yong**
+
+Here's the image of me:
+![yong](https://www.google.com/url?sa=i&source=images&cd=&ved=2ahUKEwjWi6iu___kAhUH4o8KHT8DDIkQjRx6BAgBEAQ&url=https%3A%2F%2Fwww.hiclipart.com%2Ffree-transparent-background-png-clipart-fpirq&psig=AOvVaw22sWrYd_Ie8p1e4QUUcMfN&ust=1570188830918964)
```

Comment:
Your image does not work, I would recommend you using an image with a link that ends with `.png` or `.jpg` or something similar.